### PR TITLE
Fix underscore bug in container names

### PR DESCRIPTION
**Pull request by joelea**
See test for bug.

### CODE REVIEW

**Comment by felixdesouza** [comment]
Although this works for the default cases, if you decide to set your container name for say hostname/linking reasons or whatever, it will fail to detect. Or is that coming in a separate PR?

**Comment by joelea** [comment]
Yeah, this isn't the longterm fix, just a quick fix for a super dumb bug that exists in the implementation we *wanted*. There's some longer term work to fix the bigger thing.

File: src/test/java/com/palantir/docker/compose/connection/ContainerNamesTest.java [hunk 1]
```diff
@@ -37,6 +37,12 @@ public void ps_output_with_a_single_container_results_in_a_single_container_name
     }
 
     @Test
+    public void containers_with_underscores_in_their_naem_are_allowed() {
```
Comment (jamesneale) [inline src/test/java/com/palantir/docker/compose/connection/ContainerNamesTest.java]: nit: naem -> name

Comment (joelea) [inline src/test/java/com/palantir/docker/compose/connection/ContainerNamesTest.java]: Thanks.

**Pull request merged**